src/Orleans/Orleans.csproj

```diff
@@ -140,6 +140,8 @@
     <Compile Include="Streams\PubSub\FaultedSubscriptionException.cs" />
     <Compile Include="Streams\PubSub\ImplicitStreamPubSub.cs" />
     <Compile Include="Streams\PubSub\StreamPubSubImpl.cs" />
+    <Compile Include="Streams\QueueAdapters\IQueueFlowController.cs" />
```

Comment:
I think you had some issue with merge and rebase. These changes belong to other unrelated commit.